frontend/src/static/js/v_summary.js

```diff
@@ -552,17 +552,10 @@ window.vSummary = {
             (commit) => commit.date >= tsince && commit.date <= tuntil,
         );
 
-        const commits = [];
```

Comment:
Why delete this? This should not be a part of this PR.

Reply:
My bad, we had a misunderstanding about the purpose of these few lines of code.